recipes/pauda-mergeblast/merge.blast.pl

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+# xiechaos@gmail.com
```

Comment:
Sorry I am new to this. Can you please clarify?
Do you mean download the script to recipes/pauda folder and just copy it to PREFIX/bin ?